src/view/pipeline/pipelineViewLoader.ts

```diff
@@ -0,0 +1,100 @@
+/*-----------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE file in the project root for license information.
+ *-----------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+import { Trigger } from '../../tekton/pipelinecontent';
+import { Progress } from '../../util/progress';
+import { TektonItem } from '../../tekton/tektonitem';
+import { commands } from 'vscode';
+
+export interface NameType {
+  name: string;
+  type: string;
+}
+
+export interface Sudhir {
+  name: string;
+}
+
+export interface Params {
+  default: string;
+  description: string;
+  name: string;
+}
+
+export interface Workspaces {
+  name: string;
+  workspaceName?: string;
+  workspaceType?: string;
+  key?: string;
+  value?: string;
+  subPath?: string;
+  emptyDir?: string;
+}
+
+
+export const stringToContext = new Map<string, Trigger>();
+
+export default class PipelineViewLoader {
+
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  static get extensionPath() {
+    return vscode.extensions.getExtension('redhat.vscode-tekton-pipelines').extensionPath
+  }
+
+  // eslint-disable-next-line @typescript-eslint/require-await
+  static async loadView(title: string, context: any): Promise<vscode.WebviewPanel> {
+    stringToContext.set('pipelineStart', context);
+    const localResourceRoot = vscode.Uri.file(path.join(PipelineViewLoader.extensionPath, 'out', 'pipelineView'));
+
+    const panel = vscode.window.createWebviewPanel('pipelineView', title, vscode.ViewColumn.One, {
+      enableScripts: true,
+      localResourceRoots: [localResourceRoot],
+      retainContextWhenHidden: true
+    });
+    panel.iconPath = vscode.Uri.file(path.join(PipelineViewLoader.extensionPath, 'images/tekton.svg'));
```

Comment:
This will not work for windows, this line should be ` panel.iconPath = vscode.Uri.file(path.join(PipelineViewLoader.extensionPath, 'images','tekton.svg'));`